modules/installation-vsphere-machines.adoc

```diff
@@ -0,0 +1,135 @@
+/ Module included in the following assemblies:
+//
+// * installing/installing_vsphere/installing-vsphere.adoc
+
+[id="installation-vsphere-machines-{context}"]
+= Creating {op-system-first} machines in vSphere
+
+Before you install a user provisioned infrastructure (UPI) cluster on vSphere,
+you need to create {op-system} machines on a single host for it to use.
+
+.Prerequisites
+
+* Obtain the Ignition config files for your cluster.
+* Have access to an HTTP server that you can access from your computer and that
+the machines that you create can access.
+
+.Procedure
+
+////
+This will work for GA.
+. Obtain the {op-system} images from the
+link:https://access.redhat.com/downloads[Product Downloads] page on the Red
+Hat customer portal.
+////
+
+. Convert the bootstrap, master, and worker Ingintion conf files to Base64
+encoding.
+
+. Upload the Base64-encoded bootstrap Ignition config file that the
+installation program created to your HTTP server. Note the URLs of this file.
++
+You must host the bootstrap Ignition config file because it is too large to
+store in temporary memory.
+
+. Save a secondary Ignition config file for your bootstrap node to your computer.
++
+----
+{
+  "ignition": {
+    "config": {
+      "append": [
+        {
+          "source": "<bootstrap_ignition_config_url>", <1>
+          "verification": {}
+        }
+      ]
+    },
+    "timeouts": {},
+    "version": "2.1.0"
+  },
+  "networkd": {},
+  "passwd": {},
+  "storage": {},
+  "systemd": {}
+}
+----
+<1> Specify the URL of the bootstrap Ignition config file that you hosted.
++
+When you create the Virtual Machine (VM) for the bootstrap machine, you reference
+this this Ignition config file.
+
+. Convert the temporary bootstrap file to Base64 encoding.
+
+. Obtain the {op-system} OVA image:
++
+----
+$ curl -J -L -O https://releases-art-rhcos.svc.ci.openshift.org/art/storage/releases/ootpa/410.8.20190412.1/rhcos-410.8.20190412.1-vmware.ova
+----
+
+. In the vSphere Client, create a folder in your datacenter to store your VMs.
+The folder name must match the cluster name that you specified in the
+`install-config.yaml` file.
+
+. In the vSphere Client, create a template for the OVA image.
++
+[NOTE]
+====
+In the following steps, you use the same template for all of your cluster
+machines and provide the location for the Ignition config file for that machine
+type when you provision the VMs.
+====
+.. From the *Hosts and Clusters* tab, right-click your cluster's name and
+click *Deploy OVF Template*.
+.. On the *Select an OVF* tab, specify the name of the {op-system} OVA file
+that you downloaded.
+.. On the *Select a name and folder* tab, set a *Virtual machine name*, such
+as {op-system}, and click the name of the your vSphere cluster.
+.. On the *Select a computer resource* tab, click the name of the your vSphere
+cluster.
+.. On the *Select storage* tab, configure the storage options for your VM.
+For example, select *Thin Provision* if you plan to supply
```

Comment:
I think that Thin Provision should be our recommendation. It does not depend on whether the user is going to supply additional storage.